client/src/pages/LandingPage/index.js

```diff
@@ -34,6 +32,67 @@ const useStyles = makeStyles({
 export default function LandingPage() {
     const classes = useStyles();
     const history = useHistory();
+    const [signupMsgboxOpen, setSignupMsgboxOpen] = useState(false);
+    const [signupMsg, setSignupMsg] = useState("Sign up failed, invalid Email address or password");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const fetchSignup = (username, password) => {
+        console.log(username);
+        console.log(password);
```

Comment:
remove? + all other console logs in this page

Reply:
The setToken is to store the token in localstorage through the ```use-persisted-state``` package. This is how the auth context uses it for private routing